docs/src/faq.md

```diff
@@ -146,9 +146,9 @@ given Ethereum NFT is stored.
 
 *Inscriptions are much simpler.*
 
-Ethereum NFTs depend on the Ethereum network and virtual machine, which are
-highly complex, constantly changing, and which introduce changes via
-backwards-incompatible hard forks.
+Ethereum NFTs significantly depend on the Ethereum Virtual Machine(a runtime environment on the Ethereum network) and ERC-based Smart Contracts(ERC-721, ERC-1155, ETC), which are
```

Comment:
Unless there are typos, I think the original text is good here. I might be wilin' out on twitter but I want to keep the docs here more neutral.

Reply:
I was aiming more for detailed clarity to better assert the distinction. The information provided doesn't break any form of neutrality or promote a stance of any sort, it simply educates.